bpf/kmesh/bpf2go/kernelnative/enhanced/kmeshsockops_bpfeb.go

```diff
@@ -16,34 +16,19 @@ type KmeshSockopsBuf struct{ Data [40]int8 }
 
 type KmeshSockopsClusterSockData struct{ ClusterId uint32 }
 
-type KmeshSockopsLogEvent struct {
-	Ret uint32
-	Msg [255]int8
-	_   [1]byte
+type KmeshSockopsKmeshConfig struct {
+	BpfLogLevel      uint32
+	NodeIp           [4]uint32
+	PodGateway       [4]uint32
+	AuthzOffload     uint32
+	EnableMonitoring uint32
 }
 
 type KmeshSockopsManagerKey struct {
 	NetnsCookie uint64
 	_           [8]byte
 }
 
-type KmeshSockopsRatelimitKey struct {
-	Key struct {
-		SkSkb struct {
-			Netns  uint64
-			Ipv4   uint32
-			Port   uint32
-			Family uint32
-			_      [4]byte
-		}
-	}
-}
-
-type KmeshSockopsRatelimitValue struct {
```

Comment:
Will there be any effect when this part is deleted?

Reply:
No effect，Ratelimit is not enable when kernel enhanced